keyboards/smithrune/iron180v2/v2s/info.json

```
@@ -0,0 +1,149 @@
+{
+    "keyboard_name": "Iron180 Sv2",
+    "manufacturer": "SmithRune",
+    "url": "",
+    "maintainer": "Gondolindrim",
+    "usb": {
+        "vid": "0x8384",
+        "pid": "0x1181",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["C7" , "C6" , "B15", "B14", "B13", "B12", "B10", "B3" , "D2" , "B0" , "C5" , "C12", "C10", "A10", "A15", "C0" , "C1" , "A7"],
+        "rows": ["B4" , "C11", "A8" , "C9" , "B1" , "C4" , "B5" , "B6"]
+    },
+    "features": {
+        "audio": false,
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "lto": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": false
+    },  
```

1. remove encoder definition (`false` is default)
2. LTO enabled as `build.lto`
```suggestion
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "rgblight": false
    },
    "build": {
        "lto": true
    },
```